final_project/snapvisite/forms.py

```diff
@@ -1,6 +1,6 @@
 from django import forms
 from django.forms import inlineformset_factory
-from .models import Company, Category, Address, Schedule
+from .models import *
```

Comment:
Nie używamy gwiazdki, wiem, że to działa, ale nie jest to dobra praktyka, trzeba jawnie wypisać co importujemy